### DISCUSSION TITLE
Sobre o limite na franquia de internet.

### DISCUSSION THREAD

**Discussion by vitor-mendrone-deel** [frontendbr/forum#116] [Discussão]
Acho que todos vocês já sabem o que está rolando: As teles estão impondo limites de dados em seus serviços de internet fixa e com isso já começaram a prejudicar a rotina de milhares de pessoas. 

http://www.gazetaonline.com.br/_conteudo/2016/03/noticias/dinheiro/3933346-operadoras-vao-cortar-internet-fixa-apos-o-fim-da-franquia.html

http://gizmodo.uol.com.br/operadoras-franquia-mensal-internet-fixa/

http://www.proteste.org.br/tecnologia/celular/noticia/proteste-considera-ilegal-que-operadoras-imponham-limites-a-internet-fixa

http://g1.globo.com/distrito-federal/noticia/2016/03/empresas-querem-vender-internet-fixa-como-pacote-de-dados-diz-mp-do-df.html

Esse limite vai muito além de limitar nosso acesso ao Netflix/Youtube

Essa limitação tira do alcance de todos serviços como Khan Academy, Code Academy, Udemy, EDX, Code School, WebSchool (Alou @suissa), os cursos do @willianjusten, do @fdaciuk, entre milhares de outros. 

Ela também afeta aos desenvolvedores que já não acompanham esses cursos porém utilizam a internet para se comunicar com o time (trabalhando remotamente), para downloads de programas, libs, etc, (com esse limite, aquele devDependencies forrado de coisas vai virar um pesadelo)

Esse limite coloca em cheque nossa liberdade profissional justamente por limitar o acesso ao conhecimento e todos nós sabemos que vivemos em um país onde a educação é sofrível, muitos chegam ao mercado sem saber o básico e acabam encontrando a saída nos sites que citei acima. 

Existe uma iniciativa na internet que pretende lutar contra essa imposição das teles: 

https://www.facebook.com/mislbr

Eu gostaria de saber o posicionamento de vocês sobre o assunto, se já conheciam essa iniciativa e se pretendem agir de alguma forma contra essa decisão, se sua internet já foi "cortada" por conta da franquia (alou @elvisdetona), enfim, iniciar aqui uma discussão importante (afinal de conta, nós, programadores, fazemos a internet) e quem sabe propor ideias do que podemos fazer para apoiar o movimento que já existe com algo que vá alem do compartilhamento nas redes sociais.

**Comment by felipefialho** [comment]
E quais são as nossas opções?

Eu que trabalho remoto vou ser absurdamente afetado. Sem contar a questão da diversão (Spotify, Netflix, Youtube...).

Faz um tempo que rolou umas questões com relação a alterações no Marco Civil que censurariam a internet, não me impressionaria que fora a guerra contra o Netflix, um certo interesse em desinformar o povo também esteja envolvido.

Até então, das 4 grandes (Vivo, Oi, NET e TIM), apenas a TIM não se posicionou (ainda), e por coincidência, é a única que não tem TV a cabo, porém uma migração em massa pra TIM provavelmente não vai ser permitida pelo "cartel", já que vão chorar de concorrência desleal para a Anatel ou a TIM vai subir absurdamente os preços.

Existem provedores menores que podem fugir dessa situação?

É realmente muito preocupante.

**Comment by bernardodiasc** [comment]
Algum link contendo as informações dos fatos? Não to muito por dentro dos detalhes mas tambem pra que a issue não fique assim tão vaga.

**Comment by vitor-mendrone-deel** [comment]
@bernardodiasc coloquei alguns links de referência no post original, pra evitar que fragmente. 

@LFeh Pois é, é uma luta complicada, principalmente se as grandes afetadas (Google, Facebook e Netflix) não se posicionarem (como fez o Uber)

Propus mais cedo no grupo do FSP que fizéssemos um manifesto assinado por profissionais de TI de varias empresas pra que enviássemos ao congresso e também ao Movimento Internet sem Limites, como forma de apoio. 

Acredito que a segunda parte seja fazer com que as pessoas cobrem os deputados/senadores e a própria Anatel, por e-mail, telefone, página do Facebook, Twitter, etc, em resumo realizar um trabalho de base conscientizando as pessoas sobre isso. 

A luta é política mas temos a tecnologia do nosso lado.

**Comment by baltazarparra** [comment]
Uma ótima deixa para a [Google Fiber](https://fiber.google.com/about/).

**Comment by rickbenetti** [comment]
@LFeh tem um problema com a LiveTim só privilegiados poderão migrar, a cobertura dela é muito pequena e restrita.

@baltazarparra seria uma boa se a Google viesse, mas o mercado boicotaria isso como fizeram com o SmartForTwo em 1998*.

*Na época a SmartForTwo não era da Mercedes e ela ia colocar uma fábrica no Brasil e já tinha até anunciado preços abaixo de R$ 15k e o mercado não gostou, fizeram pressão no congresso pra barrar e só permitiram a entrada como importado pra ficar mais caro, aí anos depois a Mercedes comprou, fabrica aqui boa parte, manda pra fora pra reimportar pra usar como desculpa o alto preço, ou seja um projeto com a da Google daria muita merda e por isto acredito que pra Google isso não é bom.

**Comment by vitor-mendrone-deel** [comment]
@baltazarparra @rkb81 um outro problema nisso é o fato de que o Google Fiber não atende nem mesmo todas as cidades dos EUA. 

O mesmo com a Live TIM aqui no BR, em São Paulo por exemplo ela só atende áreas nobres. 

Qualquer iniciativa de uma empresa menor que se propusesse a não limitar a franquia esbarraria na logística de cabear um país inteiro.

**Comment by fernandoadms** [comment]
Essa questão é complicada, de barrarem o tráfego de dados. Mas acredito que isto não irá ocorrer. Na minha opinião este tipo de dado foi colocado pelas operadoras na mídia para ver qual seria a opinião dos consumidores e se iriam reagir bem ou não.

Não acredito que chegaríamos ao ponto de dar um passo atrás na evolução da web. Apesar de ter certeza absoluta que isto esta acontecendo porque em meios de crise vale mais uma pessoa assinar uma excelente operadora de Internet e investir em Netflix do que pagar além da INTERNET + TV POR ASSINATURA + TELEFONE FIXO (caso da NET).

**Comment by vitor-mendrone-deel** [comment]
@FeSantos Pois é, de inicio eu achei que era um teste das teles pra ver qual seria a reação das pessoas, porém só hoje já tive 3 relatos de internet cortada por ter atingido o limite da franquia. Uma delas do @ElvisDetona ):

**Comment by rickbenetti** [comment]
@FeSantos aqui em casa é a NET (sem TV a cabo), mesmo quando tinha a TV eu já fui barrado ou limitado por ter extrapolado e muito o pacote, normalmente eu consumo 3 a 4 vezes o limite da NET.
Minha conexão já foi de 10MB, 25MB e hoje estou nos 15MB.

**Comment by baltazarparra** [comment]
Então pode ser a deixa pra alguma dessas major's mesmo, criarem outro modelo para entrega de serviços, porque com certeza, ninguém está satisfeito com isso.

**Comment by fernandoadms** [comment]
Então a situação é pior do que imaginava @Mendrone e @rkb81. 

Eu já tive muitos problemas na época que a VIVO ainda era Speddy com essa questão de barrar velocidade por limite de dados consumidos mas graças a Live Tim não tive mais esse problema.

Das poucas operadoras que tive o prazer e desprazer de usar a LIVE TIM foi a que tive a melhor experiência.

Mas atualmente estou de mudança e provavelmente para onde irei não sei se terá LIVE TIM e pode ser que ficarei refém dessas operadoras com esse pensamento retrógado. 

Não adianta eles tentarem barrar o consumo de dados porque a Internet esta evoluindo e em algum dia isso será lembrado e iremos dar risada dessa situação.

**Comment by felipefialho** [comment]
Segundo a TIM, eles não tem previsão de adicionar limite. Só não sei se da pra confiar.

![image](https://cloud.githubusercontent.com/assets/3603793/14464615/db008700-00a5-11e6-907f-390f488eab47.png)

**Comment by fernandoadms** [comment]
@LFeh tranquilo e favorável foi foda.... ahahahaha

**Comment by rickbenetti** [comment]
O ruim da LiveTim, só rico ou quase isso pra ter acesso e outra se chover pode contar de 2 a 3 dias sem, amigos que moram em regiões atendidas já me falaram disso.

**Comment by vitor-mendrone-deel** [comment]
Aqui na empresa utilizamos todas as operadoras, uma delas é a TIM que já chegou a passar quase 30 dias fora do ar, horrível, de uns meses pra cá melhorou, mas ainda cai diariamente e isso porque estamos em Moema. um dos metros quadrados mais caros do Brasil.

**Comment by vitor-mendrone-deel** [comment]
Mas voltando, a discussão que propus é tentar encontrar caminhos que não sejam correr pra outras operadoras, até porque a TIM vive na corda bamba e logo menos é comprada por outra empresa. 

O objetivo é discutirmos o que podemos fazer para ajudar o movimento existente pra barrar esse absurdo.

**Comment by lagden** [comment]
Palhaçada total...
é muita cara de pau e serviço de qualidade inferior...
eles já diminuem a velocidade na sua cara desde sempre!!

Não consigo entender o sentido de um empresário desse!!! A galera vai começar a procurar alternativas e eles irão perder mais clientes ainda!!

**Comment by kazzkiq** [comment]
> ##### **tl;dr** Oligopólio come solto com ajuda da ANATEL

Segundo dados da ANATEL, o marketshare de Banda Larga no Brasil atualmente está assim:

| Empresa/Grupo | Acessos | % Marketshare |
| --- | --- | --: |
| NET | 8.158.288 | 31,92% |
| VIVO | 7.337.537 | 28,71% |
| OI | 6.372.380 | 24,93% |
| ALGAR | 461.712 | 1,81% |
| TIM | 262.306 | 1,03% |
| _Outras_ | _~3.000.000_ | _~10%_ |

[Fonte](http://www.anatel.gov.br/dados/index.php/component/content/article?id=269)

É assustador ver que apenas NET, VIVO (Telefonica) e Oi são responsáveis por **85% do acesso a Banda Larga do país**.

Apesar de apenas três empresas serem donas da maior parte dos clientes, segundo a TELECO e a ANATEL, no Brasil existem mais de 1900 empresas fornecendo internet (a maior parte dessas atendem apenas regionalmente, por serem menores).

O maior problema ao meu ver, é o que gera o "oligopólio" NET-VIVO-Oi. Aparentemente, não é a falta de outras empresas no mercado (já que existem mais de 1900 atuando no país). Mas sim a conivência e passividade da ANATEL com as grandes empresas, a falta de incentivo para melhorar a infraestrutura da rede, e a alta burocracia (como sempre) que afeta principalmente as empresas menores.

Segundo a Sebrae, para abrir uma empresa provedora de internet no país é necessário contratar "circuitos troncos" providos pela Embratel. Como a Sebrae só cita esta opção para a infraestrutura, eu imagino que ou não existem outras formas, ou é tão caro que apenas empresas muito grandes conseguiriam pagar pela instalação (e por isso a Sebrae nem cita no site). De qualquer forma, por mais que existam inúmeras empresas menores no segmento, **a infraestrutura continua centralizada na Embratel**.

Vale lembrar que a Embratel pertence à América Móvil, empresa que **também é dona da NET e da Claro**.

Não me admira que a NET seja a maior provedora de Banda Larga do país. É como se uma empresa de trens fosse a dona das ferrovias, que dúvida que os trens dela seriam os mais beneficiados, né?!

Me admira a ANATEL (que teoricamente defende o consumidor) permitir um monopólio de infraestrutura como estes.

De qualquer forma, além dessa _sujeirada_ toda da infraestrutura, ainda tem a alta burocracia do Governo para conseguir abrir uma empresa de Banda Larga no Brasil. A Sebrae detalha o processo:

> É necessário contratar um contador profissional para legalizar a empresa nos seguintes órgãos:
> - Junta Comercial;
> - Secretaria da Receita Federal (CNPJ);
> - Secretaria Estadual de Fazenda;
> - Prefeitura Municipal, para obter o alvará de funcionamento;
> - Enquadramento na Entidade Sindical Patronal em que a empresa se enquadra (é obrigatório o recolhimento da Contribuição Sindical Patronal por ocasião da constituição da empresa e até o dia 31 de janeiro de cada ano);
> - Caixa Econômica Federal, para cadastramento no sistema “Conectividade Social – INSS/FGTS”;
> - Corpo de Bombeiros Militar.
> 
> Além do cumprimento das exigências anteriores, é necessário pesquisar na Prefeitura Municipal se a Lei de Zoneamento permite a instalação de provedor de acesso à internet.

---

Em outras palavras, pra prover internet no Brasil você precisa usar uma infraestrutura monopolizada e tendenciosa, passar por um processo burocrático para poder prover o serviço, e ainda sofrer com todos os outros problemas indiretos de infraestrutura/impostos que o país possui (falta de segurança, altos preços de luz, gasolina, veículos, impostos sob produtos, etc).

Estamos falando do Brasil, então não vejo a situação melhorando tão cedo. Enquanto isso, o jeito é deixar a Netflix de lado.

**Comment by lagden** [comment]
Para evitar isso seria via `ANATEL`, depois é `Reclame Aqui` e se não resolver é só botar fogo em tudo! 🔥

**Comment by rickbenetti** [comment]
Pena que no Brasil a Eletrobras, não faz o que a Enel está fazendo na italia: http://www.webnews.it/2016/03/23/enel-fibra-ottica-224-citta/ 224 cidades receberão este ano a super fibra, eles pretendem passar cabo em todas as cidades e fazem parcerias com empresas de internet local e celular pra utilizar a estrutura assim recuperando o investimento.

**Comment by geovanerocha** [comment]
Ridiculo, como se já não bastasse o que fazem na internet móvel.

Se isso de fato se expandir, vai ser foda, principalmente no nosso caso, que, querendo ou não, dependemos 100% de internet, já ia foder tudo no npm install.

**Comment by rickbenetti** [comment]
Só fico pensando na tragédia vc precisa rodar npm install pra testar uma correção emergêncial e no meio do processo a banda acaba e vc tem que ligar pra pedir mais que leva 48 a 96hs pra ativar.

**Comment by dleitee** [comment]
Começar a utilizar alternativas como [essa](https://addyosmani.com/blog/using-npm-offline/)

**Comment by kazzkiq** [comment]
@dleitee Uhauhauhauau só no Brasil mesmo pra dev ser forçado a usar package manager offline.

É rir pra não chorar mesmo...

**Comment by dleitee** [comment]
@kazzkiq cara isso é revoltante. 

O pior é que não temos nenhuma perspectiva de mudança.

**Comment by kazzkiq** [comment]
@dleitee Não mesmo. E isso não é novidade, a Anatel já foi acusada várias vezes de facilitar a formação de cartel entre as quatro maiores empresas de telecom.

É aquele tipo de situação que não dá mais pra reverter sem demitir todo mundo do setor e recontratar gente nova. Os funcionários atuais da Anatel devem estar tão atolados em falcatruas que mesmo que eles quisessem não dá mais pra voltar atrás porque se não a chantagem come solta.

**Comment by vitor-mendrone-deel** [comment]
FYI

https://github.com/jlcarvalho/InternetSemLimites

**Comment by DevCarlosAugusto** [comment]
https://www.change.org/p/vivo-gvt-oi-net-claro-anatel-minist%C3%A9rio-publico-federal-contra-o-limite-na-franquia-de-dados-na-banda-larga-fixa
